tests/functional/broadcast/07-timeout.t

```diff
@@ -18,5 +18,6 @@
 # Test broadcasts a timeout setting
 . "$(dirname "$0")/test_header"
 set_test_number 2
+export ABORT_ON_TASK_FAIL=true
```

Comment:
Was this test run with abort on failure before?

Reply:
Looks like this is a mistake. Well spotted.